docs/src/limitations.md

```diff
@@ -28,3 +28,13 @@ repeatedly executing code that modifies some global state
 can lead to unexpected side effects.
 Rebugger works best on methods whose behavior is determined solely by their input
 arguments.
+
+## Note for Ubuntu users
+
+The default meta key on Ubuntu is left Alt, which is equivalent to Esc Alt on the default Gnome terminal emulator. 
+
+### Ubuntu 16.04
+
+Rebugger may not work with the default keyboard shortucts on Ubuntu 16.04. The root of the issue is not solved yet. As a walkaround please re-map the meta keys to function keys, e.g. with `Rebugger.add_keybindings(stepin="\e[17~", stacktrace="\e[18~")` to map step-in to F6 and stacktrace to F7.
```

Comment:
Actually, I suspect the root of the problem is that gnome doesn't allow this to be customized: https://bugs.launchpad.net/ubuntu/+source/nautilus/+bug/1113420

Also, `add_keybindings` is appropriate if you're starting Rebugger from an existing REPL, but if you're customizing your `startup.jl` file it's better to use
```julia
    Rebugger.keybindings[:stepin] = "\e[17~"      # Add the keybinding F6 to step into a function.
    Rebugger.keybindings[:stacktrace] = "\e[18~"  # Add the keybinding F7 to capture a stacktrace.
```
as is already in the example on [this page](https://timholy.github.io/Rebugger.jl/latest/config.html#Customize-keybindings-1). The reason has to do with the order in which keybindings are added to the REPL and whether any "stale" bindings that might have side effects are still present. (Doing it before `atreplinit` means that there won't be any stale bindings.)

Reply:
It is very likely so. Strange, that the Esc-E combination does not work as well... it would mean, that the translation Alt+<key> into Esc-<key> is done in lower level and is not Julia-specific.